lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -548,6 +549,15 @@ private function translateNamespaces(Query\ResultSetMapping $rsm): void
     public function setHydrationCacheProfile(?QueryCacheProfile $profile = null)
     {
         if ($profile === null) {
+            if (func_num_args() < 1) {
+                Deprecation::trigger(
+                    'doctrine/orm',
+                    'TODO',
```

Comment:
I'm the most forgetful person in the world and I want to thank you for making me feel better about that!

Reply:
🤦🏻‍♂️